component/portal/src/main/java/org/exoplatform/portal/jdbc/entity/NodeEntity.java

```diff
@@ -82,6 +84,13 @@ public class NodeEntity implements Serializable {
   @OneToOne(fetch = FetchType.EAGER, mappedBy = "rootNode")
   private NavigationEntity  navigationEntity;
 
+  @Enumerated(EnumType.ORDINAL)
+  @Column(name = "TARGET")
+  private NodeTarget        nodeTarget;
+
+  @Column(name = "DESCRIPTION")
+  private String            description;
```

Comment:
description is not yet added as field in frontend, so IMO to be defined here later